docs/contributing/how-to-run-website-locally.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "How to run the website locally"
+linkTitle: "How to run the website locally"
+weight: 2
+slug: how-to-run-the-website-locally
+---
+
+The Cortex documentation is compiled into a website published at [cortexmetrics.io](https://cortexmetrics.io/). This document explains how to run the website locally, in order to have a quick feedback loop while contributing to the documentation or website styling.
+
+
+## Initial setup
+
+The following initial setup is required only once:
+
+1. Install [Hugo](https://gohugo.io/) `v0.59.1`
+2. Install [Node.js](https://nodejs.org/en/) (or via [`nvm`](https://github.com/nvm-sh/nvm))
```

Comment:
I think it might worth specifying a minimum node version as guidance, I had to switch from system (10.x) to one of the lastest table (12.x) for the instructions to work.